src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -141,6 +141,40 @@ describe('Step Users', () => {
         },
       };
 
+      await waitFor(() => {
+        expect(receivedRequest).toEqual(expectedRequest);
+      });
+    });
+    test('with valid name and password and check the Administrator checkbox', async () => {
+      const user = userEvent.setup();
+      await renderCreateMode();
+      await goToRegistrationStep();
+      await clickRegisterLater();
+      await goToUsersStep();
+      await addValidUser();
+      const isAdmin = screen.getByRole('checkbox', {
+        name: /administrator/i,
+      });
+      user.click(isAdmin);
+      await goToReviewStep();
+      // informational modal pops up in the first test only as it's tied
+      // to a 'imageBuilder.saveAndBuildModalSeen' variable in localStorage
+      //    await openAndDismissSaveAndBuildModal();
```

Comment:
This can be removed completely, the modal opens only on first try, which is covered in previous test.